mysql/errname.go

```diff
@@ -316,7 +316,7 @@ var MySQLErrName = map[uint16]string{
 	ErrUnknownTimeZone:                          "Unknown or incorrect time zone: '%-.64s'",
 	ErrWarnInvalidTimestamp:                     "Invalid TIMESTAMP value in column '%s' at row %d",
 	ErrInvalidCharacterString:                   "Invalid %s character string: '%.64s'",
-	ErrWarnAllowedPacketOverflowed:              "Result of %s() was larger than maxAllowedPacket (%d) - truncated",
+	ErrWarnAllowedPacketOverflowed:              "Result of %s() was larger than max_allowed_packet (%d) - truncated",
```

Comment:
This is original warning msg in mysql?

Reply:
Yes, this error message comes from the mysql-server code base.